editor/osm_editor.cpp

```diff
@@ -1250,7 +1250,7 @@ bool Editor::IsFeatureUploadedImpl(FeaturesContainer const & features, MwmSet::M
 }
 
 const char * const Editor::kPlaceDoesNotExistMessage =
-    "The place has gone or never existed. This is an auto-generated note from OMaps application: "
+    "The place has gone or never existed. This is an auto-generated note from Oragnic Maps application: "
     "a user reports a POI that is visible on a map (which can be outdated), but cannot be found on "
```

Comment:
It would be great to insert here the date of our maps snapshot. @vng can you please take a look?

Reply:
Please open a ticket instead.

Reply:
https://github.com/organicmaps/organicmaps/issues/919